examples/graphql/package.json

```diff
@@ -31,14 +31,15 @@
     "url": "https://github.com/open-telemetry/opentelemetry-js/issues"
   },
   "dependencies": {
-    "@opentelemetry/api": "^0.15.0",
-    "@opentelemetry/exporter-collector": "^0.15.0",
+    "@opentelemetry/api": "^0.17.0",
+    "@opentelemetry/exporter-collector": "^0.17.0",
+    "@opentelemetry/instrumentation": "^0.17.0",
     "@opentelemetry/instrumentation-graphql": "^0.13.0",
```

Comment:
We should either use a file reference here like in express or keep the old version. 0.13 will not work with 0.17
applies several examples